src/javascript/binary/static/cashier/paymentagent_withdrawws.js

```diff
@@ -0,0 +1,271 @@
+var PaymentAgentWithdrawWS = (function() {
+    "use strict";
+
+    var containerID = '#paymentagent_withdrawal';
+    var $views      = $(containerID + ' .viewItem');
+    var errorClass = 'errorfield';
+    var viewIDs = {
+        error   : '#viewError',
+        success : '#viewSuccess',
+        confirm : '#viewConfirm',
+        form    : '#viewForm'
+    };
+    var fieldIDs = {
+        ddlAgents : '#ddlAgents',
+        txtAmount : '#txtAmount',
+        txtDesc   : '#txtDescription'
+    };
+    var formData, isValid;
+    var withdrawCurrency = 'USD',
+        minAmount = 10,
+        maxAmount = 2000;
+
```

Comment:
Maybe we will change these values (In fact, if no problem, I have changed the values in my branch')
So it will be great if we get these values from server directly.